MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1299,95 +1227,90 @@ private IEnumerable<string> SplitArtists(string val, char[] delimiters, bool spl
         /// <param name="info">The info.</param>
         /// <param name="tags">The tags.</param>
         /// <param name="tagName">Name of the tag.</param>
-        private void FetchStudios(MediaInfo info, Dictionary<string, string> tags, string tagName)
+        private void FetchStudios(MediaInfo info, IReadOnlyDictionary<string, string> tags, string tagName)
         {
-            var val = FFProbeHelpers.GetDictionaryValue(tags, tagName);
+            var val = tags.GetValueOrDefault(tagName);
 
-            if (!string.IsNullOrEmpty(val))
+            if (string.IsNullOrEmpty(val))
             {
-                var studios = Split(val, true);
-                var studioList = new List<string>();
+                return;
+            }
 
-                foreach (var studio in studios)
-                {
-                    // Sometimes the artist name is listed here, account for that
-                    if (info.Artists.Contains(studio, StringComparer.OrdinalIgnoreCase))
-                    {
-                        continue;
-                    }
+            var studios = Split(val, true);
+            var studioList = new List<string>();
 
-                    if (info.AlbumArtists.Contains(studio, StringComparer.OrdinalIgnoreCase))
-                    {
-                        continue;
-                    }
+            foreach (var studio in studios)
+            {
+                // Sometimes the artist name is listed here, account for that
+                if (info.Artists.Contains(studio, StringComparer.OrdinalIgnoreCase))
+                {
+                    continue;
+                }
 
-                    studioList.Add(studio);
+                if (info.AlbumArtists.Contains(studio, StringComparer.OrdinalIgnoreCase))
+                {
+                    continue;
                 }
 
-                info.Studios = studioList
-                    .Where(i => !string.IsNullOrWhiteSpace(i))
-                    .Distinct(StringComparer.OrdinalIgnoreCase)
-                    .ToArray();
+                studioList.Add(studio);
             }
+
+            info.Studios = studioList
+                .Where(i => !string.IsNullOrWhiteSpace(i))
```

Comment:
Could do this at the beginning of the foreach loop, no?

Reply:
Yep. Changed it. Instead of the `Distinct`, we could event change it to use a Set, right?